src/pretix/presale/checkoutflow.py

```diff
@@ -412,6 +413,9 @@ class QuestionsStep(QuestionsViewMixin, CartMixin, TemplateFlowStep):
     identifier = "questions"
     template_name = "pretixpresale/event/checkout_questions.html"
     label = pgettext_lazy('checkoutflow', 'Your information')
+    contact_overrides = False
+    contact_overrides_initial = {}
+    contact_overrides_disabled = {}
```

Comment:
This is generally a bad idea. The way properties work in python, this will be **the same dict** with shared content across all requests handled on this Django thread, so state can leak between requests. These 3 variables should be initialized in ``__init__``